.circleci/src/workflows/web_beta_delivery.yml

```diff
@@ -0,0 +1,68 @@
+when:
```

Comment:
As this workflow and the `web_production_delivery` workflow are almost the same, i think if would be good to merge them in one file and use conditionals, where they differ (only two lines)

Reply:
AFAIK it is not possible to use conditionals in workflows, has something changed here?